### PR TITLE
Fixed footer and sidebar issue

### PR DESCRIPTION
**Pull request by MonikaSharma06**
Fixed footer and sidebar issue:
- Footer issue: Now it is showing at the bottom of the page even when the content is short (in Settings and Service Health)
- Fixed the gap in the left side of sidebar

### CODE REVIEW

File: src/frontend/src/app/footer/footer.component.html [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <footer class="footer">
   <div class="container">
-    <div class="content has-text-centered">
+    <div class="content has-text-centered has-text-grey-dark">
       <p>
```
Comment (praveen-garg) [inline src/frontend/src/app/footer/footer.component.html]: instead of p use small tag

**Comment by shariq1501** [comment]
navbar
screen shot 2017-12-18 at 2 42 31 pm
 overlap issue while scrolling page content.

File: src/frontend/src/app/audit-log/audit-log.component.html [hunk 2]
```diff
@@ -1,5 +1,81 @@
...
+            </td>
+          </tr>
+        </tbody>
+      </table>
+      <div *ngIf='totalPagesCount > 1'>
+        <app-pagination [count]="recordCount" [page]="page" [perPage]="limit" [totalPage]='totalPagesCount' (goPrev)="onPrev()" (goNext)="onNext()"
+          (goFirst)="onFirst()" (goLast)="onLast()" (goPage)="goToPage($event)">
+        </app-pagination>
+      </div>
+
```
Comment (shariq1501) [inline src/frontend/src/app/audit-log/audit-log.component.html]: remove extra space here.

**Comment by praveen-garg** [comment]
Good work @monika-Dianomic, Thanks!

**Pull request merged**